### DISCUSSION TITLE
No matching version found for @refinedev/core@^4.5.0

### DISCUSSION THREAD

**Discussion by mxn2020** [refinedev/refine#3940] [Q&A] [answered]
Hello everyone,

I get following error trying to install the mern-dashboard-client sample:

npm ERR! code ETARGET
npm ERR! notarget No matching version found for @refinedev/core@^4.5.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

The server was installed and runs just fine.

Thanks!

**Comment by omeraplak** [comment, marked as answer]
Hey,
Thank you for contacting us. Could you try again now?

**Comment by mxn2020** [comment]
This was fast! Worked fine. Thank you very much!


I found your tutorial and entire project while searching for a solution on filters for mongodb on react. Struggling with that big time. Will check your example app now and probably post some questions on that matter on this forum, would that be okay?
